<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KbdStateConfig</class>
 <widget class="QDialog" name="KbdStateConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="leds">
     <property name="title">
      <string>Leds</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="showCaps">
        <property name="text">
         <string>Show Caps Lock</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showNum">
        <property name="text">
         <string>Show Num Lock</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showScroll">
        <property name="text">
         <string>Show Scroll Lock</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="switchMode">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="showLayout">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Show keyboard layout</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="showFlags">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Show flags instead labels</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Layout mode:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="switchGlobal">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Global</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">modes</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="switchWindow">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Window</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">modes</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="switchApplication">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Application</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">modes</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="configureLayouts">
        <property name="text">
         <string>Configure layouts</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="btns">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btns</sender>
   <signal>accepted()</signal>
   <receiver>KbdStateConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btns</sender>
   <signal>rejected()</signal>
   <receiver>KbdStateConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="modes"/>
 </buttongroups>
</ui>
